src/Workspaces/Remote/ServiceHub/Host/ChecksumSynchronizer.cs

```diff
@@ -49,8 +49,8 @@ public async ValueTask SynchronizeSolutionAssetsAsync(Checksum solutionChecksum,
                 {
                     using var _ = PooledHashSet<Checksum>.GetInstance(out var checksums);
 
+                    checksums.Add(solutionCompilationChecksumObject.SourceGeneratorExecutionVersionMap);
                     solutionChecksumObject.AddAllTo(checksums);
-                    checksums.Remove(solutionChecksumObject.Checksum);
```

Comment:
>checksums.Remove(solutionChecksumObject.Checksum); [](http://example.com/codeflow?start=20&length=50)

Just want to make sure this is an intentional change

Reply:
yeah.  it basically doesn't hurt to have this in the set.  the asset provider will basically see immediately that it has it and not fetch it.  so i just wanted to siplify this code.